deployment/ansible/group_vars/all

```diff
@@ -19,17 +19,17 @@ postgresql_username: mmw
 postgresql_password: mmw
 postgresql_database: mmw
 
-postgresql_version: "9.6"
-postgresql_package_version: "9.6.*.pgdg16.04+1"
+postgresql_version: "12"
+postgresql_package_version: "12.*.pgdg20.04+1"
 postgresql_support_repository_channel: "main"
-postgresql_support_libpq_version: "13.*.pgdg16.04+1"
-postgresql_support_psycopg2_version: "2.7"
-postgis_version: "2.3"
-postgis_package_version: "2.3.*.pgdg16.04+1"
+postgresql_support_libpq_version: "13.*.pgdg20.04+1"
+postgresql_support_psycopg2_version: "2.7.7"
+postgis_version: "3"
+postgis_package_version: "3.1.*.pgdg20.04+1"
```

Comment:
Consider how this aligns with the version of PostgreSQL and PostGIS we plan to target on RDS. Also, consider that we're installing a different version of `libpq` (the PostgresSQL C library) from the client tooling we're using.

In the past, we've pinned these packages pretty broadly. We should be able to copy the work done in https://github.com/azavea/docker-postgis/pull/28 to add the [PGDG archive APT repository](https://www.postgresql.org/about/news/announcing-apt-archivepostgresqlorg-2024/) and stay pinned to the same versions that RDS will use.

Reply:
Thanks for pointing me to this resource! Fixed in 92b5748.